tutorials/knapsack_problem.py

```diff
@@ -265,6 +265,7 @@ def __init__(self, knapsack_packages: List[KnapsackPackage], name: str = 'genera
         self.knapsack_packages = knapsack_packages
         DessiaObject.__init__(self, name=name)
 
+    @markdown_view('markdown display')
```

Comment:
you can rename this, eg: Knapsack markdown

Reply:
```suggestion
    @markdown_view('Generator markdown')
```